src/CitizenProject/CitizenProjectManager.php

```diff
@@ -344,6 +344,26 @@ public function removeImage(CitizenProject $citizenProject): void
         $citizenProject->setImageUploaded(false);
     }
 
+    public function changePrivilege(Adherent $adherent, CitizenProject $citizenProject, string $privilege, bool $flush = true): void
```

Comment:
It's confusing to have 2 almost the same functions. The function with the same name, almost the same list of arguments already exists https://github.com/EnMarche/en-marche.fr/blob/d5a32b4a5fddb28b585aeb058ef9dac67681c522/src/CitizenProject/CitizenProjectAuthority.php#L37
We should have only one `changePrivilege ` function.